backend/app/elasticsearch/connect.py

```diff
@@ -0,0 +1,81 @@
+from elasticsearch import Elasticsearch
+
+
+# To confirm if it can connect to elasticsearch serevr
+def connect_elasticsearch():
+    _es = None
+    _es = Elasticsearch("http://localhost:9200")
+    if _es.ping():
+        print("Successfully connected to Elasticsearch")
+    else:
+        print("Can not connect to Elasticsearch")
+    return _es
+
+
+# Create index
+def create_index(es_object, index_name):
+    created = False
+    # index settings
+    settings = {
+        "settings": {"number_of_shards": 3, "number_of_replicas": 3},
```

Comment:
same for the number of shards and number of replicas, could we move them to `config.py`